source/module_cell/module_paw/paw_cell_libpaw.cpp

```diff
@@ -223,7 +224,7 @@ void Paw_Cell::set_libpaw_files()
     filename_list = new char[ntypat*264];
     if(GlobalV::MY_RANK == 0)
     {
-        std::ifstream ifa(GlobalV::stru_file.c_str(), std::ios::in);
+        std::ifstream ifa(PARAM.inp.stru_file.c_str(), std::ios::in);
```

Comment:
PARAM.inp.stru_file  and GlobalV::stru_file are different?

Reply:
GlobalV::Stru_file and PARAM.inp.stru_file are the same,but GlobalV::Stru_file will change during the program in the below lines:
`    if (PARAM.inp.calculation == "md" && PARAM.mdp.md_restart) // md restart  liuyu add 2023-04-12
    {
        int istep = 0;
        double temperature = 0.0;
        MD_func::current_md_info(GlobalV::MY_RANK, PARAM.globalv.global_readin_dir, istep, temperature);
        if (PARAM.inp.read_file_dir == "auto")
        {
            GlobalV::stru_file = PARAM.globalv.global_stru_dir + "STRU_MD_" + std::to_string(istep);
        }`
in fact, it will change and can't be asserted in the input file.